### PR TITLE
Feature/swagger UI

### PR DESCRIPTION
**Pull request by dpnolte**
Hi,

This PR introduces support for swagger-ui.
The support is contained in the module `kompendium-swagger-ui`.
The Swagger UI webjar together with Ktor's Webjars feature is used. I've added a section to the readme how to enable Swagger UI.

I've also added the following in this PR:
- I fixed issue when using unit to indicate no params or empty response body. Swagger was complaining about this. When providing Unit, the content was still filled and you got a ref like `#/components/schemas/Unit` which didn't exist. I've also added a test for this case
- I've set the `jvmTarget` to 11. Java 11 is still the current latest LTS. If it is set to 14, projects using java 11 cannot use Kompendium. This would exclude many projects like mine. You can still compile / run this with jdk > 11 (also 14). I really really hope you find this change okay 🙏

### CODE REVIEW

**Comment by rgbrizzlehizzle** [comment]
> I've set the jvmTarget to 11

Totally fine with me, all my internal projects are running 14 but makes sense to keep it at LTS... Perhaps this should even support 8? Since so much of the community is still on 8.  Anyway, issue for another day 😄

File: kompendium-core/src/test/kotlin/org/leafygreens/kompendium/KompendiumTest.kt [hunk 1]
```diff
@@ -486,6 +503,16 @@ internal class KompendiumTest {
     }
   }
 
+  private fun Application.emptyGet() {
+    routing {
+      route("/test/empty") {
+        notarizedGet<Unit, Unit>(emptyTestGetInfo) {
```
Comment (rgbrizzlehizzle) [inline kompendium-core/src/test/kotlin/org/leafygreens/kompendium/KompendiumTest.kt:509]: Hmm strange that this didn't work, I thought I had squashed this bug already 😅 Thanks for fixing

**Review by rgbrizzlehizzle — APPROVED** [review]
looks great 🔥

**Comment by dpnolte** [comment]
Thanks for merging 💯

**Pull request merged**